src/shapes/Textbox.ts

```diff
@@ -256,11 +258,26 @@ export class Textbox extends IText {
    * @param {Number} desiredWidth width you want to wrap to
    * @returns {Array} Array of lines
    */
-  _wrapText(lines: Array<any>, desiredWidth: number): Array<any> {
-    const wrapped = [];
+  _wrapText(lines: string[], desiredWidth: number, reservedSpace = 0) {
+    const wrapped: string[][] = [];
     this.isWrapping = true;
+    const { lines: data, minWidth } = this.measureLinesForWrapping(lines);
```

Comment:
We should cache this and recalc only when text/styles change.
This will make resizing a lot faster